crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -90,6 +90,17 @@ def addons
       end
     end
 
+    def repocheck
+      {
+        cloud: {
+          available: repo_version_available("suse-openstack-cloud", "7") == 0
```

Comment:
Again, does this check make sense in master branch? We want to run the check before the upgrade, to check if the _new_ repos are in place. So in master branch the check should be for cloud8